ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -440,7 +507,17 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
         guard let newWindow else {
             return
         }
+
         tokenSet.update(newWindow.fluentTheme)
+        titleView.tokenSet.update(newWindow.fluentTheme)
```

Comment:
`AvatarTitleView` already implements `willMove(toWindow: )`, do you really need to update the theme here when it's already doing it?

Reply:
Looks like we don't need it.